src/docs/development/ui/advanced/splash-screen.md

```diff
@@ -115,8 +115,59 @@ to the normal theme at the appropriate time.
 The Android app now displays the desired launch screen
 while the app initializes.
 
+### Android S
+
+See [Android Splash Screens][] first on how to configure your splash screen on
+Android S.
+
+Make sure neither `io.flutter.embedding.android.SplashScreenDrawable` is set in
+your manifest, nor is `provideSplashScreen` implemented. Doing so will cause
+the Android splash screen to fade smoothly into the Flutter when the app is
+launched.
+
+Some apps may want to continue showing the last frame of the Android splash
+screen in Flutter, to preserve the illusion of a single frame to users while
+additional loading continues in Dart. To achieve this, the following Android
+APIs may be helpful:
+
+```java
+import android.os.Build;
+import android.window.SplashScreenView;
+import androidx.core.view.WindowCompat;
+
+public class MyFlutterActivity extends FlutterActivity {
+    @Override
+    protected void onCreate(Bundle savedInstance) {
+        // ...
+
+        // Aligns the Flutter view vertically with the window.
+        WindowCompat.setDecorFitsSystemWindows(getWindow(), false);
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+            // Disable the Android splash screen fade out animation to avoid
+            // a flicker before the similar frame is drawn in Flutter.
+            getSplashScreen()
+                .setOnExitAnimationListener(
+                    (SplashScreenView splashScreenView) -> {
+                        splashScreenView.remove();
+                    });
+        }
+    }
+}
+```
+
```

Comment:
Also, this needs changes to AndroidManifest.xml

Reply:
Are you referring to how users should not be using `io.flutter.embedding.android.SplashScreenDrawable` here? I don't think we need to mention it here, since this section is more for how to do the alignment of the logo

Reply:
I was referring to modifying`<activity>` tag